platform-api/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceService.java

```diff
@@ -190,7 +182,7 @@ public UsersWorkspaceDto getByName(@PathParam("name") String name) throws Server
         //TODO add maxItems & skipCount to manager
         return workspaceManager.getRuntimeWorkspaces(getCurrentUserId())
                                .stream()
-                               .map(this::asDto)
+                               .map(this::asRuntimeWorkspaceDto)
```

Comment:
The same [comment](https://github.com/codenvy/che-core/pull/307#discussion_r41075354)